.goreleaser.yaml

```
@@ -11,6 +11,12 @@ builds:
     id: "runner"
     binary: "runner"
     main: ./go/src/wel/commands/runner/main/runner.go
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm64
```

To be consistent with our current released binaries we should also add `386`, unless you don't think we no longer need it.

```suggestion
    goarch:
      - amd64
      - arm64
      - 386
```